customer/src/test/java/no/unit/nva/customer/create/CreateChannelClaimHandlerTest.java

```diff
@@ -107,16 +115,25 @@ void shouldReturnForbiddenWhenCreatingChannelClaimWithWrongAccessRight() throws
     }
 
     @Test
-    void shouldReturnBadRequestWhenInvalidChannelUriIsProvidedInRequest() throws IOException {
-        var request = createRequestWithInvalidChannelUriInBody();
+    void shouldReturnConflictWhenClaimingChannelAlreadyClaimedByCustomer() throws IOException {
+        var request = createRequestClaimingChannelAlreadyClaimedByCustomer();
         handler.handleRequest(request, outputStream, context);
         var response = GatewayResponse.fromOutputStream(outputStream, Problem.class);
-        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, response.getStatusCode());
+        assertEquals(HttpURLConnection.HTTP_CONFLICT, response.getStatusCode());
+    }
+
+    @Test
+    void shouldReturnConflictWhenClaimingChannelAlreadyClaimedByAnotherCustomer()
```

Comment:
Good, I like the symmetry of saying any claim that is already claimed returns the same response.